### PR TITLE
Drop support for Symfony < 4.3

### PR DESCRIPTION
**Pull request by wbloszyk**
## Subject

Drop support for Symfony < 4.3.

I am targeting this branch, because this change respect BC.

### CODE REVIEW

File: src/BasketBundle/Controller/BasketController.php [hunk 1]
```diff
@@ -34,7 +35,7 @@
 /**
  * This controller manages the Basket operation and most of the order process.
  */
-class BasketController extends Controller
+class BasketController extends AbstractController
```
Comment (jordisala1991) [inline src/BasketBundle/Controller/BasketController.php:38]: This look like a BC break

**Comment by jordisala1991** [comment]
Can you fill the PR template?

File: src/BasketBundle/Controller/BasketController.php (hunk 1 above)
Comment (wbloszyk) [reply src/BasketBundle/Controller/BasketController.php:38]: I think i should do it in master

**Comment by wbloszyk** [comment]
Should i drop symfony 4.3 too?

**Comment by jordisala1991** [comment]
Is it worth it?

**Comment by wbloszyk** [comment]
Symfony 4.3 is not maintained. I think we can keep support for 4.3 and drop in in master.

**Comment by jordisala1991** [comment]
That would be good I think. Make all BC changes on 3.x. Upgrade to 4.4 and bc break on master

**Comment by jordisala1991** [comment]
Thank you @wbloszyk

**Comment by wbloszyk** [comment]
@core23 I know but diffrence is not do much. Also all sonata 3 bundles support it. It will be good idea to keep minimum require symfony 4.3 in whole sonata 3.

**Comment by core23** [comment]
What I wanted to say is that our release cycle is very slow for this bundle. IMHO you can already create a new PR to drop 4.3.

**Comment by wbloszyk** [comment]
I will focus on sandbox and PageBundle now. I needed in to my 3 commerce project. I have many improve in my roadmap for this bundle too. I will update some issue on next week like sonata 4, blocks upgrade, sandbox 3 and 4 roadmap, ecommerce.

**Comment by greg0ire** [comment]
Should this be patch or minor?

**Comment by wbloszyk** [comment]
Next release should be minor and should include https://github.com/sonata-project/ecommerce/pull/678. This PR is RTM. Will be awesome release it with drop CoreBundle (both drop `twig/extensions`).

**Pull request merged**